### DISCUSSION TITLE
Integrating Amasty OSC

### DISCUSSION THREAD

**Discussion by IbrahimS2** [breezefront/community#290] [1. Help]
From what I remember, you said that Breeze does not support the checkout, does that mean we do not have to integrate it as we normally do?

I am using the public docs:
![image](https://github.com/breezefront/community/assets/19901858/107ca3f0-ecfc-4dc5-b7d3-7cac1b95cf76)

**Comment by vovayatsyuk** [comment]
Yes, if you’ll watch page sources on the checkout page, you’ll see requirejs. 

Docs saying about checkout/cart page.

**Reply by IbrahimS2** [reply]
@vovayatsyuk But Amasty OSC is not working/misbehaving, any clue?

**Reply by vovayatsyuk** [reply]
What do you mean by "not working"?

**Reply by IbrahimS2** [reply]
I just sent the module via WeTransfer, give it a try

**Reply by IbrahimS2** [reply]
@vovayatsyuk Did it work for you as expected? the layout is messed up even though I mapped the relevant CSS etc...

**Reply by vovayatsyuk** [reply]
Yes. Everything looks nice.

![image](https://github.com/breezefront/community/assets/306080/f8e30887-0e3f-4c82-bd4b-b2daf2e6d15a)

**Reply by IbrahimS2** [reply]
@vovayatsyuk What is the css mapping you have done if I may ask? it never worked for me.

**Reply by vovayatsyuk** [reply]
No, I just enabled the module.

**Reply by vovayatsyuk** [reply]
~~Maybe it took some styles from firecheckout. I have it installed too.~~

**Reply by IbrahimS2** [reply]
Oh that is why, it never worked for me and just using Amasty OSC

**Reply by vovayatsyuk** [reply]
Just checked again. It works fine. Styles are taken from the external file `/frontend/Swissup/breeze-evolution/en_US/Amasty_CheckoutCore/css/source/mkcss/amcheckout.css`

**Reply by IbrahimS2** [reply]
@vovayatsyuk Not working either, I am using the modern layout and this is what I am getting 
![image](https://github.com/breezefront/community/assets/19901858/e5eb58e4-efff-4546-8289-71051871f583)


No errors in the console either.

**Reply by IbrahimS2** [reply]
@vovayatsyuk Any clue?

**Reply by vovayatsyuk** [reply]
As I said before, after the initial install, everything working fine. Their CSS is added on the page and everything looks fine. Maybe "modern" design requires Luma theme. Try using the default design instead.

**Reply by IbrahimS2** [reply]
@vovayatsyuk I will arrange an integration so it can work using a modern and classic layout too.
